contents/blog/best-hipaa-compliant-ab-testing-tools.md

```diff
@@ -0,0 +1,129 @@
+
+#### Kameleoon and HIPAA compliance
+
+- **Self-Hosting:** No
+- **BAA Available**: Yes
+
+Kameleoon is a cloud-only service, but it will sign a BAA for use in healthcare products.
+
+What [Kameleoon says](https://www.kameleoon.com/en/health-insurance-portability-and-accountability-act-hipaa):
+
+> "Our technology platform is designed to meet the Act’s requirements in areas such as password expiry rules, secure data transfer and automatic logouts. We follow clear, transparent processes in how we handle and protect data and will sign Business Associate Agreements (BAA) as part of any agreement with clients."
+
+#### How much does Kameleoon cost?
+
+Kameleoon doesn't publish pricing publicly, but conversion optimization consultants BrillMark [state](https://www.brillmark.com/kameleoon-ab-testing-platform/#:~:text=The%20yearly%20licensing%20pricing%20for,pay%20for%20the%20annual%20license) pricing starts at $35,000 per year and scales based on depending on traffic volume.    
+
+### VWO
+
+![vwo testing](../images/blog/hipaa-compliant-ab-testing/vwo-testing.png)
+
+VWO offers a range of different products. Its testing product, VWO Testing, includes visual and code editors. You can run both A/B and multivariate tests, though the latter is limited to its Pro and Enterprise plans.
+
+#### VWO and HIPAA compliance
+
+- **Self-Hosting:** No
+- **BAA Available**: Yes
+
+VWO is a cloud-hosted platform only and will enter into a BAA with its users.
+
+What [VWO says](https://vwo.com/compliance/hipaa/):
+
+>"VWO customers who are subject to HIPAA and wish to use the VWO products with PHI must sign a BAA with VWO. Customers are responsible for ensuring that they achieve compliance with HIPAA and HITECH Act requirements. 
+>
+>"We adhere to the HIPAA obligations by leveraging appropriate security configuration options for all VWO products. Additionally, we make our Business Associate Agreement (BAA) available for execution by subscribers."
+
+#### How much does VWO cost?
+
+VWO doesn't publish its pricing publicly, but it does offer a 14-day free trial.
+
+> **HIPAA password requirements:** HIPAA requires Covered Entities and Business Associates to have "creating, changing, and safeguarding passwords". Kameleoon and VWO state this means you must force users to update their passwords every 90 days, but this is based on [out of date advice](https://www.hipaajournal.com/what-are-the-hipaa-password-expiration-requirements/) first published in 2004. [Updated advice](https://www.hipaajournal.com/what-are-the-hipaa-password-expiration-requirements/) published by the National Institute of Standards and Technology in 2017 states "Verifiers SHOULD NOT require memorized secrets [passwords] to be changed arbitrarily (e.g., periodically)."  
```

Comment:
Think I'm going to move this into an FAQ at the bottom of the article and boxout feels weird, but I don't want it in the intro section either.